src/main/java/de/hpi/bpt/argos/eventProcessing/status/EntityStatusCalculatorImpl.java

```diff
@@ -1,10 +1,7 @@
 package de.hpi.bpt.argos.eventProcessing.status;
 
-import de.hpi.bpt.argos.api.entity.EntityEndpoint;
 import de.hpi.bpt.argos.eventProcessing.EventReceiver;
 import de.hpi.bpt.argos.eventProcessing.mapping.EventEntityMappingStatus;
-import de.hpi.bpt.argos.storage.PersistenceAdapterImpl;
-import de.hpi.bpt.argos.storage.dataModel.entity.Entity;
 import de.hpi.bpt.argos.storage.dataModel.mapping.EventEntityMapping;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'de.hpi.bpt.argos.storage.dataModel.mapping.EventEntityMapping'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)